services/task-svc/migrations/000001_init.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE IF NOT EXISTS tasks (
+	id uuid PRIMARY KEY DEFAULT uuid_generate_v4(),
+	name text NOT NULL,
+	description text  NOT NULL DEFAULT '',
+    status integer NOT NULL,
```

Comment:
Formatting, also the brackets

Also an integer is not a great way to store this. We should either use an enum, a string or separate tables.

Reply:
there is some kind of formatting error here

```suggestion
       status integer NOT NULL,
```